agent/consul/internal_endpoint.go

```diff
@@ -168,7 +168,16 @@ func (m *Internal) ServiceTopology(args *structs.ServiceSpecificRequest, reply *
 		&args.QueryOptions,
 		&reply.QueryMeta,
 		func(ws memdb.WatchSet, state *state.Store) error {
-			index, topology, err := state.ServiceTopology(ws, args.Datacenter, args.ServiceName, &args.EnterpriseMeta)
+			authz, err := m.srv.ResolveToken("")
```

Comment:
Can this happen outside of the loop? I thought that acl default policy isn't hot reloadable?

Reply:
Removed this bit, see above